tests/main/aspects/task.yaml

```diff
@@ -0,0 +1,74 @@
+summary: Check the aspects API
+
+# ubuntu-14.04-64's curl doesn't support --unix-socket
+systems: [-ubuntu-14.04-*]
+
+prepare: |
+  snap install test-snapd-curl --edge --devmode
+  snap alias test-snapd-curl.curl curl
```

Comment:
Is the alias required?

Reply:
No but it shortens the lines, making the test a bit more readable